### PR TITLE
wagl_nrt: update wagl image (change in dataset yaml format)

### PR DESCRIPTION
**Pull request by uchchwhash**
- simplify logic when there is nothing to be done

### CODE REVIEW

File: dags/infra/images.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 S3_TO_RDS_IMAGE = "538673716275.dkr.ecr.ap-southeast-2.amazonaws.com/geoscienceaustralia/s3-to-rds:0.1.4"
 
 WAGL_IMAGE = (
-    "538673716275.dkr.ecr.ap-southeast-2.amazonaws.com/dev/wagl:patch-20210422-3"
+    "538673716275.dkr.ecr.ap-southeast-2.amazonaws.com/dev/wagl:patch-20210423-1"
```
Comment (NikitaGandhi) [inline dags/infra/images.py:18]: How are we publishing this wagl image? any why it is `dev/wagl` ?

Comment (uchchwhash) [reply dags/infra/images.py:18]: it's always been, I have been pushing it here.

Comment (NikitaGandhi) [reply dags/infra/images.py:18]: I feel as we moving forward to production ready, we better follow some practice :)

Comment (uchchwhash) [reply dags/infra/images.py:18]: It has got a binary version of MODTRAN in there, so we cannot really put it to dockerhub or anywhere public.

Comment (uchchwhash) [reply dags/infra/images.py:18]: Also this is a patched overlayed version of the original docker image that runs in production which we cannot rebuild. We are working on replacing it with a rebuildable version.

Comment (NikitaGandhi) [reply dags/infra/images.py:18]: ECR is also equivalent to store image in dockerhub. Also don't want to stop this PR but please consider ECR repository naming convention and publishing pipeline. Create a JIRA backlog to capture this.

**Review by NikitaGandhi — APPROVED** [review]

**Pull request merged**